features/karaf/framework/src/main/resources/resources/bin/oh2_dir_layout

```diff
@@ -3,7 +3,9 @@
 # DIRNAME is the directory of karaf, setenv, etc.
 (unset CDPATH) >/dev/null 2>&1 && unset CDPATH
 
-export OPENHAB_HOME=`cd "$DIRNAME/.."; pwd`
+if [ -z ${OPENHAB_CONF} ]; then
```

Comment:
Not sure if this condition is fine cause it makes `OPENHAB_HOME` depending on setting `OPENHAB_CONF`. Is that's right?

Reply:
Crap - you're right - this `if` should be `if [ -z ${OPENHAB_HOME} ]; then` - stupid cut and paste error.